src/Symfony/Component/Cache/Tests/Simple/AbstractRedisCacheTest.php

```diff
@@ -37,9 +37,10 @@ public static function setUpBeforeClass(): void
         if (!\extension_loaded('redis')) {
             self::markTestSkipped('Extension redis required.');
         }
-        if (!@((new \Redis())->connect(getenv('REDIS_HOST')))) {
-            $e = error_get_last();
-            self::markTestSkipped($e['message']);
+        try {
+            (new \Redis())->connect(getenv('REDIS_HOST'));
```

Comment:
Same here

Reply:
There's a check for the extension few lines above.